pkg/cmd/roachtest/run.go

```diff
@@ -51,6 +56,13 @@ const (
 	testResultFailure testResult = iota
 	testResultSuccess
 	testResultSkip
+
+	// bucket and secretsLocation are for fetching the creds from store
+	bucket            = "roachtest-data"
```

Comment:
It looks like this bucket is in the project `cockroach-ephemeral`. Can you please use Terraform to create a bucket in `e2e-infra` or another project that is wholly owned and managed by TestEng?

Reply:
The reason I kept is here is to make it accessible from everyone laptop and gceworker.